bitcoin/bitcoin.cabal

```diff
@@ -5,7 +5,7 @@ cabal-version: 1.12
 -- see: https://github.com/sol/hpack
 
 name:           bitcoin
-version:        0.1.0
+version:        0.1.1
```

Comment:
nit: Should this not be a major version bump? I'd also be okay with not bumping the version because we haven't really made an official release yet, have we?

Reply:
Definitely. This is a mistake. Originally I was going to avoid changing the `bitcoin` api altogether but as this evolved there are tiny changes to it. I'll bump to 0.2.0. I don't mind aggressively bumping version numbers to ensure that we don't forget to do so.